query/parser.go

```diff
@@ -108,7 +108,9 @@ func parseDate(date string) (int64, error) {
 	for _, format := range dateFormats {
 		t, err := time.Parse(format, date)
 		if err == nil {
-			return t.Unix(), nil
+			// Multiply the number of seconds by 1000 to convert to milliseconds,
```

Comment:
You don't have to be so specific with the comments. Something like `Convert to millis` or just a top-level comment on `parseDate` saying return value is in timestamp millis would be sufficient (preferably latter).

It's a bit of an art to figure out where to comment and how much to, so it might take a while to get the hang of.